CONTRIBUTING.md

```diff
@@ -17,39 +18,50 @@ If you are contributing to the project for the first time, please consider check
 In case you get stuck, please feel free to ask for help in the [Discord](https://discord.com/invite/U2peSNf23P) server or GitHub Discussions.
 
 ## Setup the Project Locally
```

Comment:
This section is to setup `intro` locally, and not guestbook. Guestbook doeesn't required to run `docsify`.
But is this what we want in this guide?

Reply:
Oh yeah I forgot about that @adiati98 🤦‍♀️

Reply:
Hmm... We have a getting started section in the intro course to fork and clone repo.

Reply:
True @adiati98. Maybe we can put a link to that section? It could serve as a refresher. :)

Reply:
But would love to hear your idea about how to modify it.

Reply:
With our adjustments, we want the contributing guide to only be about contributing to improve this repository, not how to contribute to the guestbook. It's worth saying something like this:

> This CONTRIBUTING guide will walk you through how to contribute to this project for bug fixes and features. If you want to contribute to the guestbook, please refer to chapter five of the intro course.

Reply:
@BekahHW, we're now in the guestbook repository. 🙂

Reply:
@adiati98, right. The contributing guide for the guestbook should only include information about contributing to improve the repository. For directions on how to add yourself, that would be found in the intro course, right? 

I'm thinking that maybe another option would be like we do at Virtual Coffee with issue 13 (members adding themselves); we have a pinned issue here to add yourself to the repo, but the issue links to the chapter in the course.

Reply:
@BekahHW we can do that. Add an open issue, and contributors can write `Relates to #000` instead of `Closes #000`. I think we're doing that as well in `pizza-verse`.

The problem with that (based on experience with VC) is sometimes, we need to reopen the issue because contributors who quite understands how OSS works use `closes` or `fixes` instead of mentioning the issue.

Another way is, we have a point to find or create an issue in the [Getting Started With Contributing](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=getting-started-with-contributing). I think it would be great to put it into practice in this guestbook repo with these reasons:

- Beginners learn how to open an issue (with templates).
- They learn how to link a PR with an issue and can experience the issue & PR close automatically.

Reply:
I second this